skimage/transform/hough_transform.py

```diff
@@ -148,6 +148,27 @@ def hough_circle(image, radius, normalize=True, full_output=False):
         Hough transform accumulator for each radius.
         R designates the larger radius if full_output is True.
         Otherwise, R = 0.
+
+    Examples
+    --------
+    >>> from skimage.transform import hough_circle
+    >>> img = np.zeros((100,100),dtype=np.bool_)
+    >>> X,Y = np.meshgrid(np.arange(100),np.arange(100))
+    >>> x0,y0,radius = 20,35,23
+    >>> circle = np.abs((X-x0)**2+(Y-y0)**2-radius**2)<5**2
```

Comment:
I'm not sure I understand how you define the circle, isn't it ``circle = X-x0)**2+(Y-y0)**2 < radius**2``?